.github/workflows/tests.yaml

```diff
@@ -0,0 +1,35 @@
+name: "Tests"
+
+on: [ push ]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        php: [ '7.1', '7.2', '7.3', '7.4', '8.0' ]
+    name: PHP ${{ matrix.php }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: xdebug
```

Comment:
Is xdebug needed when code coverage is not being recorded (for now) ?

Reply:
Probably not, but ideally it would be used to get coverage working! The travis version wasn't using xdebug for coverage, but I'm not sure how to configure it.

Reply:
pcov is how you get coverage in 2021 not xdebug